cypress/e2e/right-panel/file-panel.spec.ts

```diff
@@ -70,6 +70,16 @@ describe("FilePanel", () => {
     });
 
     describe("render", () => {
+        it("should render empty state", () => {
+            // Wait until the information about the empty state is rendered
+            cy.get(".mx_FilePanel_empty").should("exist");
+
+            // Take a snapshot of empty FilePanel
+            cy.get(".mx_FilePanel").percySnapshotElement("File Panel - empty", {
+                widths: [264], // Emulate the UI. The value is based on minWidth specified on MainSplit.tsx
```

Comment:
The value is available here: https://github.com/matrix-org/matrix-react-sdk/blob/de16d347e9294f9f4771ca4bc950fcf1e8c1acb4/src/components/structures/MainSplit.tsx#L74